src/layouts/components/FooterLink.tsx

```diff
@@ -35,9 +38,24 @@ export default function FooterLink({
           className={classNames(
             "text-dark-1000 font-semibold hover:accent-dfc-gradient-text cursor-pointer w-[133px] md:w-full"
           )}
+          onMouseEnter={() => setIsHovered(true)}
+          onMouseLeave={() => setIsHovered(false)}
         >
-          <Link href={url} passHref data-testid={testId}>
+          <Link
+            href={url}
+            passHref
+            data-testid={testId}
+            className="flex flex-row items-center"
+          >
             {t(`footer.sitemap.${category}.${altLabel}`)}
+            {hasUpArrowIcon ? (
```

Comment:
correct me if i am wrong, i thought link that has up arrow should use `<FooterExternalLink>`?

Reply:
But for the `White Paper` section, it is not an external link & needed an up arrow (according to the figma design), which was why I included the up arrow in the `FooterLink` component too!